### DISCUSSION TITLE
Does Schneegans have a discord or a way to contact?

### DISCUSSION THREAD

**Discussion by Arch-druid** [Schneegans/Fly-Pie#276] [Fly-Pie Q&A] [answered]
I have been absolutely loving the pie menu! I was working with blender one day and thought that a radial menu for the basic desktop would be an awesome idea, and this performs far beyond my expectations. I wanted to reach out to @Schneegans for some insight on how he developed this app and what is the roadmap for future developments. I can code C++ and python, although my javascript is very deficient. It would be great to have a discussion with you, Simon. If you are on discord, would you let me know if you have a channel or server that I can discuss some of your workflow in developing the flypie menu? Thanks in advance for your time. Great program! I converted my entire environment from KDE to GNOME simply on the basis of the FlyPIE menu alone.

**Comment by Schneegans** [comment, marked as answer]
Hi there! You can either contact me via email (code@simonschneegans.de), via matrix chat (@schneegans:matrix.org) or on Ko-fi (https://ko-fi.com/schneegans). Ko-fi however requires that you become a supporter :wink:
